libs/application/templates/new-primary-school/src/forms/Prerequisites.ts

```diff
@@ -0,0 +1,141 @@
+                newPrimarySchoolMessages.pre.nationalRegistryInformationTitle,
+              subTitle:
+                newPrimarySchoolMessages.pre
+                  .nationalRegistryInformationSubTitle,
+            }),
+            buildDataProviderItem({
+              provider: ChildrenCustodyInformationApi,
+              title: '',
+              subTitle: '',
+            }),
+            buildDataProviderItem({
+              provider: UserProfileApi,
+              title: newPrimarySchoolMessages.pre.userProfileInformationTitle,
+              subTitle:
+                newPrimarySchoolMessages.pre.userProfileInformationSubTitle,
+            }),
+          ],
+        }),
+        buildMultiField({
+          id: 'hasNoChildren',
+          title: newPrimarySchoolMessages.pre.noChildrenFoundTitle,
+
+          children: [
+            buildDescriptionField({
+              id: 'noChildrenDecsription',
+              title: '',
+              description: newPrimarySchoolMessages.pre.noChildrenFoundReasons,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenNationalRegistryDescription',
+              title: '',
+              marginTop: 3,
+              description:
+                newPrimarySchoolMessages.pre
+                  .noChildrenFoundNationalRegistryDescription,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenIslandisInfo',
+              title: '',
+              marginTop: 2,
+              description:
+                newPrimarySchoolMessages.pre.noChildrenFoundIslandisDescription,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenNationalRegistryInfo',
+              title: '',
+              marginTop: 3,
+              description:
+                newPrimarySchoolMessages.pre
+                  .noChildrenFoundNationalRegistryInfo,
+            }),
+            buildDescriptionField({
+              id: 'noChildrenMMSInfo',
+              title: '',
+              marginTop: 3,
+              marginBottom: 6,
+              description: newPrimarySchoolMessages.pre.noChildrenFoundMMSInfo,
+            }),
+          ],
+        }),
+      ],
+    }),
+    buildSection({
+      id: 'childrenNParentsSection',
+      title: newPrimarySchoolMessages.childrenNParents.sectionTitle,
+      children: [],
+    }),
+    buildSection({
+      id: 'primarySchoolSection',
+      title: newPrimarySchoolMessages.primarySchool.sectionTitle,
+      children: [],
+    }),
+    buildSection({
+      id: 'differentNeedsSection',
+      title: newPrimarySchoolMessages.differentNeeds.sectionTitle,
+      children: [],
+    }),
+    buildSection({
+      id: 'overviewSection',
+      title: newPrimarySchoolMessages.overview.sectionTitle,
+      children: [],
+    }),
+    buildSection({
+      id: 'conclusionSection',
+      title: newPrimarySchoolMessages.conclusion.sectionTitle,
+      children: [],
+    }),
+  ],
```

Comment:
Maybe skip the formstepper in the prereq form instead of faking it?

Reply:
We have been doing this in all our applications, to show the user all steps in the application when in prereq.
Is it a new approach/rule to not show it in the prereq that we haven't heard about or ?

Reply:
Fixed.
Removed "dummy" steps from Pre
Also removed "dummy" Gagnaöflun  steps from Draft